pkgs/build-support/fetchrepoproject/default.nix

```diff
@@ -18,19 +39,21 @@ in
 
 stdenv.mkDerivation {
   buildCommand = ''
-    mkdir ./.repo
+    mkdir .repo
+    ${optionalString (local_manifests != []) ''
     mkdir ./.repo/local_manifests
-    for local_manifest in ${concatMapStringsSep " " toString localManifests}
+    for local_manifest in ${concatMapStringsSep " " toString local_manifests}
+
     do
       cp $local_manifest ./.repo/local_manifests/$(stripHash $local_manifest; echo $strippedName)
```

Comment:
I think technically a path with a space in `name` can exist in Nix. And the user can create it easily. I don't think NixPkgs is anywhere close to being space-clean, though…